tzrec/acc/trt_utils.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) 2024, Alibaba Group;
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Any, Dict, List, Optional, Sequence, Tuple, Union
+
+import torch
+import torch_tensorrt
+from torch import nn
+from torch.profiler import ProfilerActivity, profile, record_function
+from torchrec.fx import symbolic_trace
+
+from tzrec.acc.utils import is_debug_trt
+from tzrec.models.model import ScriptWrapper
+from tzrec.utils.logging_util import logger
+
+
+def trt_convert(
```

Comment:
add how to use trt in export doc

Reply:
There are still some issues, and I will add documentation in the following PR.